lib/ae_mdw/sync/async_tasks/derive_aex9_presence.ex

```diff
@@ -48,4 +50,29 @@ defmodule AeMdw.Sync.AsyncTasks.DeriveAex9Presence do
 
     :ok
   end
+
+  @spec mutations(args :: list()) :: [Mutation.t()]
+  def mutations([contract_pk, kbi, mbi, create_txi]) do
```

Comment:
As the `mutations/1` for both async tasks differ from `process/1` only on the returned value and whether the mutation is committed or not, what about the `process` call `mutations`?

Reply:
Done